widget/tabcontainer.go

```diff
@@ -26,13 +26,25 @@ func NewTabItemWithIcon(text string, icon fyne.Resource, content fyne.CanvasObje
 	return &TabItem{Text: text, Icon: icon, Content: content}
 }
 
+// TabLocation ist the location where the tabs of a tab container should be rendered
+type TabLocation int
+
+// TabLocation values
+const (
+	TabLocationTop TabLocation = iota
+	TabLocationLeft
```

Comment:
In fyne.TextAlign we use Leading and Trailing instead of Left and Right - that means we can mirror the layout when implementing Right-to-Left support.
I feel that the same may apply here and an app that uses a left hand tab position probably would want it to be right hand if the language were read in the opposite direction?